From 7f22341ceaed7603119d9d4fdbb25c28f2470637 Mon Sep 17 00:00:00 2001
From: Hyunsu Cho <phcho@nvidia.com>
Date: Mon, 12 Jan 2026 22:37:59 -0800
Subject: [PATCH 2/4] Remove nvidia-nccl-cu12 from pyproject.toml

---
 python-package/pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python-package/pyproject.toml b/python-package/pyproject.toml
index 01f2b4894..6de114260 100644
--- a/python-package/pyproject.toml
+++ b/python-package/pyproject.toml
@@ -32,7 +32,6 @@ classifiers = [
 dependencies = [
     "numpy",
     "scipy",
-    "nvidia-nccl-cu12 ; platform_system == 'Linux'",
 ]
 
 [project.urls]
-- 
2.54.0

